### DISCUSSION TITLE
Multiple DSC Systems

### DISCUSSION THREAD

**Discussion by whiteas2** [Dilbert66/esphome-dsckeybus#93] [General]
Hi,

I currently have two different alarm panels in use. One is DSC connected to HA using the Keybus interface. I plan to purchase a second DSC alarm panel to replace the other panel. Before I purchase the new panel and build a second interface I just wanted to check that HA will be able to handle 2 instances of a DSC panel?

**Comment by Dilbert66** [comment]
Sure as long as each instance has a different name

**Comment by whiteas2** [comment]
Thank you!
